accounts-journey/presentation/viewmodel/AccountsListViewModel.swift

```diff
@@ -39,7 +45,18 @@ final class AccountsListViewModel: NSObject {
         }
     }
     
-    func getAccountSummary(fromEvent event: AccountListScreenEvent) {
+    private func publishObservabilityEvents(for event: AccountListScreenEvent) {
+        switch event {
+        case .getAccounts:
+            observabilityTracker?.publish(event: AccountsJourney.Tracker.event(forScreen: .accounts))
```

Comment:
Actually this is not the screen event, semantically it's a bit wrong to call the event getAccounts.